tests/integration/integration_nodejs_test.go

```diff
@@ -92,35 +92,50 @@ func TestProjectMain(t *testing.T) {
 	}
 	integration.ProgramTest(t, &test)
 
-	t.Run("Error_AbsolutePath", func(t *testing.T) {
+	t.Run("AbsolutePath", func(t *testing.T) {
 		e := ptesting.NewEnvironment(t)
 		defer func() {
 			if !t.Failed() {
 				e.DeleteEnvironment()
 			}
 		}()
 		e.ImportDirectory("project_main_abs")
+
+		// write a new Pulumi.yaml using the absolute path of the environment as "main"
+		yamlPath := filepath.Join(e.RootPath, "Pulumi.Yaml")
```

Comment:
Total nit - but I'm surprised this works without the file bring `Pulumi.yaml`?

Reply:
Yup, that is a typo. I must have a case insensitive file system. Updated.